airflow/providers/openlineage/CHANGELOG.rst

```diff
@@ -39,17 +39,26 @@ Features
 * ``Introduce AirflowJobFacet and AirflowStateRunFacet (#39520)``
 * ``Use UUIDv7 for OpenLineage runIds (#39889)``
 * ``openlineage: execute extraction and message sending in separate process (#40078)``
+* ``Add few removed Task properties in AirflowRunFacet (#40371)``
 
 Bug Fixes
 ~~~~~~~~~
 
 * ``openlineage, redshift: do not call DB for schemas below Airflow 2.10 (#40197)``
 * ``fix: scheduler crashing with OL provider on airflow standalone (#40353)``
+* ``Revert "fix: scheduler crashing with OL provider on airflow standalone (#40353)" (#40402)``
+* ``fix: scheduler crashing with OL provider on airflow standalone (#40459)``
```

Comment:
You can move both to the hidden part of the change log

Reply:
Right. Fixed.